src/py21cmfast/src/InputParameters.c

```diff
@@ -18,72 +18,77 @@ CosmoParams *cosmo_params_global;
 AstroParams *astro_params_global;
 FlagOptions *flag_options_global;
 
+// These need to be removed, I am commenting below roughly where/when
+//  they are used, and my tentative decision on what to do with them
 GlobalParams global_params = {
-    .ALPHA_UVB = 5.0,
-    .EVOLVE_DENSITY_LINEARLY = 0,
-    .SMOOTH_EVOLVED_DENSITY_FIELD = 0,
-    .R_smooth_density = 0.2,
-    .HII_ROUND_ERR = 1e-5,
-    .FIND_BUBBLE_ALGORITHM = 2,
-    .N_POISSON = 5,
-    .T_USE_VELOCITIES = 1,
-    .MAX_DVDR = 0.2,
-    .DELTA_R_HII_FACTOR = 1.1,
-    .DELTA_R_FACTOR = 1.1,
-    .HII_FILTER = 0,
-    .INITIAL_REDSHIFT = 300.,
-    .R_OVERLAP_FACTOR = 1.,
-    .DELTA_CRIT_MODE = 1,
-    .HALO_FILTER = 0,
-    .OPTIMIZE = 0,
-    .OPTIMIZE_MIN_MASS = 1e11,
-
-
-    .CRIT_DENS_TRANSITION = 1.2,
-    .MIN_DENSITY_LOW_LIMIT = 9e-8,
+    .ALPHA_UVB = 5.0, // IonisationBox, for Gamma (move to AstroParams)
+    .EVOLVE_DENSITY_LINEARLY = 0, //PerturnbedField, uses linear growth instead of 1/2LPT (move to UserParams)
+    .SMOOTH_EVOLVED_DENSITY_FIELD = 0, //PerturbedField, smooth field after perturb (delete)
+    .R_smooth_density = 0.2, //With above, radius for smoothing (delete)
+    .HII_ROUND_ERR = 1e-5, // IonisationBox, Nion threshold to run the ES (move to define)
+    .FIND_BUBBLE_ALGORITHM = 2, // IonisationBox central pixel vs sphere (move to UserParams)
+    .MAX_DVDR = 0.2, //BrightnessTemp, maximum velocity gradient for RSDS (move to AstroParams?)
+    .DELTA_R_HII_FACTOR = 1.1, //IonisationBox, radius factor for reion ES (move to AstroParams)
+    .DELTA_R_FACTOR = 1.1, //HaloField, radius factor for DexM halo ES (move to UserParams)
 
+    .HII_FILTER = 0, //IonisationBox, filter for reion ES (move to AstroParams)
```

Comment:
I think anything that is not a continuous parameter should be in flag options, not astro params